### DISCUSSION TITLE
希望加入一些功能

### DISCUSSION THREAD

**Discussion by Redmomn** [bot-ssttkkl/nonebot-plugin-pixivbot#167] [Ideas]
希望加一个自动撤回的功能，
还有就是希望bot发送合并消息的时候发送者为发出指令的人
例如A发送了指令，bot响应之后发送了一个合并消息，合并消息中的发送人是A

**Comment by lintonxue00** [comment]
希望加入指定群号，在命令的后缀可以添加指定群号的命令，这样可以和bot的QQ私聊操作，不用在群里面操作这些指令

**Comment by lintonxue00** [comment]
举个栗子pixivbot schedule ranking 22:00 week_rookie 1-30 qq=-1 qun=12345678 qq是单独QQ号发送，qun是群号发送。-1是没有，我是参考那个ELF_RSS 订阅器的。
